examples/seismic/viscoelastic/viscoelastic_example.py

```diff
@@ -45,8 +45,10 @@ def test_viscoelastic():
 if __name__ == "__main__":
     description = ("Example script for a set of viscoelastic operators.")
     parser = ArgumentParser(description=description)
-    parser.add_argument('--2d', dest='dim2', default=False, action='store_true',
-                        help="Preset to determine the physical problem setup")
+    parser.add_argument("-nd", dest='ndim', default=3, type=int,
+                        help="Preset to determine the number of dimensions")
+    parser.add_argument("-d", "--shape", default=(150, 150, 150), type=int, nargs="+",
+                        help="Determine the grid shape, defaults in (153, 153, 153)")
```

Comment:
this is not consistent, you write the defaults here but not in the other examples. I think you can safely drop it, the --help should display it already

Reply:
Fixed